includes/utilities.inc

```diff
@@ -505,13 +505,13 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
 
     $updated_display_values
       = "<span class='toggle-wrapper'>" .
-      t("<span>!value !separator !show_more</span>", array(
+      t("<span>!value !show_more</span>", array(
         '!separator' => $separator,
         '!value' => $truncated_value,
         '!show_more' => ($value_is_truncated ? "<a href='#' class='toggler'>Show more</a>" : ""),
       ));
     if ($value_is_truncated) {
-      $updated_display_values .= t("<span>!original_value !separator<a href='#' class='toggler'>Show less</a></span>", array(
+      $updated_display_values .= t("<span>!original_value <a href='#' class='toggler'>Show less</a></span>", array(
         '!separator' => $separator,
```

Comment:
I have the same question as above about whether this change is needed? I'll try and really test this out today or tomorrow, I might just be missing something.